raw_map/src/lib.rs

```diff
@@ -268,40 +268,6 @@ impl RawMap {
         }
     }
 
-    /// Returns the corrected (but untrimmed) center and total width for a road
-    pub fn untrimmed_road_geometry(&self, id: OriginalRoad) -> Result<(PolyLine, Distance)> {
-        let road = &self.roads[&id];
-        let lane_specs = get_lane_specs_ltr(&road.osm_tags, &self.config);
-        let mut total_width = Distance::ZERO;
-        let mut sidewalk_right = None;
-        let mut sidewalk_left = None;
-        for l in &lane_specs {
-            total_width += l.width;
```

Comment:
Note this was previously wrong. We do this shifting of the OSM center line based on sidewalks on one or both sides of the road, but we didn't apply `scale_width` here to the per-lane widths.